### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by Ginsway**

### CODE REVIEW

File: src/main/resources/assets/angelica/lang/zh_CN.lang [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+sodium.option_impact.low=低
+sodium.option_impact.medium=中
+sodium.option_impact.high=高
+sodium.option_impact.extreme=极高
+sodium.option_impact.varies=视情况而定
+sodium.options.pages.quality=质量
+sodium.options.pages.performance=性能
+sodium.options.pages.advanced=高级
+sodium.options.view_distance.tooltip=能见度 - 控制地形的渲染距离。距离越短，地形渲染越少，帧率（FPS）提升越明显。
+sodium.options.simulation_distance.tooltip=模拟距离 - 控制地形和实体加载和更新的距离。更短的距离可以减少内部服务器的负载并可以提高帧率（TPS）。
```
Comment (Pistonight) [inline src/main/resources/assets/angelica/lang/zh_CN.lang:13]: FPS?

Comment (Kasualix) [reply src/main/resources/assets/angelica/lang/zh_CN.lang:13]: 提高TPS吧，帧率和括号直接删掉就行。

**Comment by mitchej123** [comment]
Is this good to merge?

**Comment by Pistonight** [comment]
> Is this good to merge?

I think TPS is a typo and it should be FPS, otherwise it looks good to me

File: src/main/resources/assets/angelica/lang/zh_CN.lang (hunk 1 above)
Comment (mitchej123) [reply src/main/resources/assets/angelica/lang/zh_CN.lang:13]:
```suggestion
sodium.options.simulation_distance.tooltip=模拟距离 - 控制地形和实体加载和更新的距离。更短的距离可以减少内部服务器的负载并可以提高帧率（FPS）。
```

**Review by mitchej123 — APPROVED** [review]

**Pull request merged**